src/web/app/components/copy-course-modal/copy-course-modal.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { StatusMessageService } from '../../../services/status-message.service';
+import { TimezoneService } from '../../../services/timezone.service';
+import { FeedbackSession } from '../../../types/api-output';
+import { COURSE_ID_MAX_LENGTH } from '../../../types/field-validator';
+
+interface Timezone {
+  id: string;
+  offset: string;
+}
+
+const formatTwoDigits: Function = (n: number): string => {
```

Comment:
With ES2017, we can do 
`const zeroPad = (num, places) => String(num).padStart(2, '0')`
[source](https://stackoverflow.com/questions/2998784/how-to-output-numbers-with-leading-zeros-in-javascript)